.github/workflows/release.yml

```diff
@@ -100,6 +100,9 @@ jobs:
         with:
           name: build-artifact
 
+      - name: Generate LICENSE files
+        run: make clone-license
```

Comment:
So it wasn't being added in make prepublish? or removed?

Reply:
The problem was that we are running `make prepublish` in a different job, and to reduce CI times we only store these files: https://github.com/babel/babel/blob/8e9143f06f72c1e3651677ed65fe94a89b8d471a/.github/workflows/release.yml#L45-L49

Reply:
We can add the cloned license to build artifacts then. It won't affect development build since they are not generated.

Reply:
The reason I split the jobs is because the "Publish release on npm" job has access to our npm token, so I wanted to run as few things as possible in it.

Currently it's only running `yarn release-tool publish --yes`, which means that it only runs code bundled ahead of time in this repository.

I will add `LICENSE` files to the artifact.